tests/UtestTest.cpp

```diff
@@ -149,6 +149,19 @@ TEST(UtestShell, TestStopsAfterSetupFailure)
     LONGS_EQUAL(0, stopAfterFailure);
 }
 
+class defaultUtestShell: public UtestShell
+{
+};
+
+TEST(UtestShell, coverageForDefaultMethods)
+{
+    // This test covers UtestShell::createTest() and Utest::testBody()
```

Comment:
Could you change the test name to include this knowledge instead of using a comment?